app/containers/Commit.tsx

```diff
@@ -1,8 +1,56 @@
 import * as React from "react";
+import { Col, Row } from "react-bootstrap";
+import { CommitFundsDescription } from "../components/CommitFundsDescription";
+import CommitFundsForm from "../components/CommitFundsForm";
 import { CommitNavbar } from "../components/CommitNavbar";
+import { UserAddressComponent } from "../components/UserAddressComponent";
+import * as style from "./Commit.scss";
 
 export const Commit: React.SFC = () => {
-  return <CommitNavbar>Commit funds in Neufund Commitment Opportunity</CommitNavbar>;
+  return (
```

Comment:
please extract whole body to separate dumb component taking all hardcoded values like 0x6ddfa40a2631348c2bd4b0c949ade1712b44587641c3309c5e8d2b914151ed50 as props. And in this component just use the newly created one pushing all hardcoded values. This way we can easily connect redux to this component. You can name newly created component as "CommitComponent" and keep it in this file. default export will be exporting connected (hardcoded for now) component

Reply:
fixed